src/main/java/hudson/plugins/spotinst/cloud/BaseSpotinstCloud.java

```diff
@@ -643,6 +737,53 @@ protected Integer getPendingThreshold() {
     protected Integer getSlaveOfflineThreshold() {
         return Constants.SLAVE_OFFLINE_THRESHOLD_IN_MINUTES;
     }
+
+    public void handleGroupManagedByOtherController() {
+        handleInitializingExpired();
+
+        if (groupAcquiringDetails.getState().equals(SPOTINST_CLOUD_COMMUNICATION_READY)) {
```

Comment:
groupAcquiringDetails.getState == comunicationState.Ready (you dont need that full string, it included in the enum name

Reply:
done